### PR TITLE
Add persona failed notifications

### PR DESCRIPTION
**Pull request by verdefred**
## Notes

image
image
image

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -62,3 +62,9 @@ en:
     talent_application_rejected_notification:
       title: "Talent application rejected"
       body: "Your talent application has been rejected by %{name}"
+    user_names_verification_failed_notification:
+      title: "Something's wrong 🤔"
```
Comment (RubenSousaDinis) [inline config/locales/en.yml:66]: No mention to verification in the app notification?

Comment (verdefred) [reply config/locales/en.yml:66]: I think you're right. I've talked with Sara and we'll change to "Verification names don't match"

File: app/services/with_persona/failed_inquiry.rb [hunk 2]
```diff
@@ -8,7 +8,12 @@ def initialize(talent:)
 
     def call
       talent.update(with_persona_id: nil)
-      # failed notification
+      CreateNotification.new.call(
```
Comment (RubenSousaDinis) [inline app/services/with_persona/failed_inquiry.rb:11]: Test the notification service is called?

File: app/services/with_persona/approve_inquiry.rb [hunk 3]
```diff
@@ -14,7 +14,12 @@ def call
         Tasks::Update.new.call(type: "Tasks::Verified", user: user)
       else
         talent.update(with_persona_id: nil)
-        # failed notification
+        CreateNotification.new.call(
```
Comment (RubenSousaDinis) [inline app/services/with_persona/approve_inquiry.rb:17]: Test the notification service is called?

**Review by RubenSousaDinis — APPROVED** [review]

**Pull request merged**